charts/thymus/templates/mailslurper.yaml

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (c) 2024 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/carbynestack/thymus.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
```

Comment:
The Mailslurper deployment should be optional, eventually coupled with user creation, not?